modules/gltf/src/lib/extensions/KHR_techniques_webgl.js

```diff
@@ -0,0 +1,58 @@
+// GLTF EXTENSION: KHR_techniques_webgl
+// https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_techniques_webgl
+/* global TextDecoder */
+import GLTFScenegraph from '../gltf-scenegraph';
+import {KHR_TECHNIQUES_WEBGL} from '../gltf-constants';
+
+function resolveTechniques({programs = [], shaders = [], techniques = []}, gltfScenegraph) {
```

Comment:
Nit: I usually put the exported function at the top and helper functions below.

Reply:
ok